components/bl-drawer-component/styles/index.less

```diff
@@ -0,0 +1,85 @@
+@bl-customComponent-drawer-themeColor: @themePrimary;
+@bl-customComponent-drawer-backgroundColor: @appBackgroundColor;
+@bl-customComponent-drawer-textColor: @appTextColor;
+@bl-customComponent-drawer-shadowColor: @appComponentShadowColor;
+@bl-customComponent-drawer-borderRadius: @appComponentBorderRadius;
+
+@bl-customComponent-drawer-container-backdrop-backgroundColor: #00000080;
+@bl-customComponent-drawer-container-transitionTimingFunction: linear;
+@bl-customComponent-drawer-container-transitionProperty: background;
+@bl-customComponent-drawer-container-zIndex: 999;
+
+@bl-customComponent-drawer-content-backgroundColor: @bl-customComponent-drawer-backgroundColor;
+@bl-customComponent-drawer-content-transitionProperty: transform;
+@bl-customComponent-drawer-content-transitionTimingFunction: linear;
+
+@bl-customComponent-drawer-button-lineHeight: 1.75;
+@bl-customComponent-drawer-button-width: 100%;
+@bl-customComponent-drawer-button-minWidth: 64px;
+@bl-customComponent-drawer-button-fontSize: 14px;
+@bl-customComponent-drawer-button-borderWidth: 0;
+@bl-customComponent-drawer-button-borderRadius: @bl-customComponent-drawer-borderRadius;
+@bl-customComponent-drawer-button-margin: 20px 10px;
+@bl-customComponent-drawer-button-padding: 6px 16px;
+@bl-customComponent-drawer-button-backgroundColor: @bl-customComponent-drawer-themeColor;
+@bl-customComponent-drawer-button-color: contrast(@bl-customComponent-drawer-button-backgroundColor);
+@bl-customComponent-drawer-button-shadowColor: @bl-customComponent-drawer-shadowColor;
+@bl-customComponent-drawer-button-shadowHover: 0px 2px 4px -1px fade(@bl-customComponent-drawer-button-shadowColor, 20%), 0px 4px 5px 0px fade(@bl-customComponent-drawer-button-shadowColor, 14%), 0px 1px 10px 0px fade(@bl-customComponent-drawer-button-shadowColor, 12%);
+
+.bl-customComponent-drawer {
+  display: flex;
+  background: @bl-customComponent-drawer-backgroundColor;
+
+  button {
+    position: relative;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    font-size: @bl-customComponent-drawer-button-fontSize;
+    width: @bl-customComponent-drawer-button-width;
+    line-height: @bl-customComponent-drawer-button-lineHeight;
+    min-width: @bl-customComponent-drawer-button-minWidth;
+    text-decoration: none;
+    padding: @bl-customComponent-drawer-button-padding;
+    background: @bl-customComponent-drawer-button-backgroundColor;
+    color: @bl-customComponent-drawer-button-color;
+    margin: @bl-customComponent-drawer-button-margin;
+    border-width: @bl-customComponent-drawer-button-borderWidth;
+    border-radius: @bl-customComponent-drawer-button-borderRadius;
+    outline: none;
```

Comment:
If you disable/hide/erase something - it will be good to add the possibility for the user easily enable it. So I think properties like this should be added to the variables list.  It's abould:
- text-decoration
- outline
- user-select
- cursor